packages/components/inputs/localized-money-input/src/localized-money-input.tsx

```diff
@@ -276,12 +391,15 @@ LocalizedMoneyInput.defaultProps = {
   horizontalConstraint: 'scale',
 };
 
-LocalizedMoneyInput.convertToMoneyValues = (values, locale) =>
+LocalizedMoneyInput.convertToMoneyValues = (values: TValue[], locale: string) =>
```

Comment:
Nit: might be good to be explicit in the return type.

```suggestion
LocalizedMoneyInput.convertToMoneyValues = (values: TValue[], locale: string): TMoneyValue[] =>
```